guides/v2.1/cloud/project/project-conf-files_magento-app.md

```diff
@@ -272,73 +308,71 @@ runtime:
-*	[ioncube](https://www.ioncube.com/loaders.php){:target="\_blank"}
-*	[pecl-http](https://pecl.php.net/package/pecl_http){:target="\_blank"}
-*	[pinba](http://pinba.org){:target="\_blank"}
-*	[propro](https://pecl.php.net/package/propro){:target="\_blank"}
-*	[pspell](http://php.net/manual/en/book.pspell.php){:target="\_blank"}
-*	[raphf](https://pecl.php.net/package/raphf){:target="\_blank"}
-*	[readline](http://php.net/manual/en/book.readline.php){:target="\_blank"}
-*	[recode](http://php.net/manual/en/book.recode.php){:target="\_blank"}
-*	[snmp](http://php.net/manual/en/book.snmp.php){:target="\_blank"}
-*	[sqlite3](http://php.net/manual/en/book.sqlite3.php){:target="\_blank"}
-*	[ssh2](http://php.net/manual/en/book.ssh2.php){:target="\_blank"}
-*	[tidy](http://php.net/manual/en/book.tidy.php){:target="\_blank"}
-*	[xcache](https://xcache.lighttpd.net){:target="\_blank"}
-*	[xdebug](https://xdebug.org){:target="\_blank"}
-*	[xhprof](http://php.net/manual/en/book.xhprof.php){:target="\_blank"}
-*	[xmlrpc](http://php.net/manual/en/book.xmlrpc.php){:target="\_blank"}
+Optional PHP extensions available to install:
+
+-  [apcu](http://php.net/manual/en/book.apcu.php){:target="\_blank"}
+-  [blackfire](https://blackfire.io/docs/up-and-running/installation){:target="\_blank"}
+-  [enchant](http://php.net/manual/en/book.enchant.php){:target="\_blank"}
+-  [gearman](http://php.net/manual/en/book.gearman.php){:target="\_blank"}
+-  [geoip](http://php.net/manual/en/book.geoip.php){:target="\_blank"}
+-  [imap](http://php.net/manual/en/book.imap.php){:target="\_blank"}
+-  [ioncube](https://www.ioncube.com/loaders.php){:target="\_blank"}
+-  [pecl-http](https://pecl.php.net/package/pecl_http){:target="\_blank"}
+-  [pinba](http://pinba.org){:target="\_blank"}
+-  [propro](https://pecl.php.net/package/propro){:target="\_blank"}
+-  [pspell](http://php.net/manual/en/book.pspell.php){:target="\_blank"}
+-  [raphf](https://pecl.php.net/package/raphf){:target="\_blank"}
+-  [readline](http://php.net/manual/en/book.readline.php){:target="\_blank"}
+-  [recode](http://php.net/manual/en/book.recode.php){:target="\_blank"}
+-  [snmp](http://php.net/manual/en/book.snmp.php){:target="\_blank"}
+-  [sqlite3](http://php.net/manual/en/book.sqlite3.php){:target="\_blank"}
+-  [ssh2](http://php.net/manual/en/book.ssh2.php){:target="\_blank"}
+-  [tidy](http://php.net/manual/en/book.tidy.php){:target="\_blank"}
+-  [xcache](https://xcache.lighttpd.net){:target="\_blank"}
+-  [xdebug](https://xdebug.org){:target="\_blank"}
+-  [xhprof](http://php.net/manual/en/book.xhprof.php){:target="\_blank"}
+-  [xmlrpc](http://php.net/manual/en/book.xmlrpc.php){:target="\_blank"}
 
 {% include note.html type="info" content="Important: PHP compiled with debug is not supported and the Probe may conflict with XDebug or XHProf. Disable those extensions when enabling the Probe. The Probe conflicts with some PHP extensions like Pinba or IonCube." %}
```

Comment:
This was missed or lost. Need to change the note tag to {:.bs-callout .bs-callout-info}